scripts/build-probe-binaries

```diff
@@ -484,6 +491,12 @@ if [ -z "$KERNEL_TYPE" ]; then
 		rhel_build $URL
 	done
 
+	#
+	# Fedora Atomic build
+	#
+	echo "Building Fedora Atomic in container"
```

Comment:
too much logging? :)

Reply:
yes a bit. this logging here was intended to express the decision that modules for fedora atomic will be built as containerized way, before calling the containerized build method (which also has its logging). I'll remove that logging message.